docs/library/uio.rst

```diff
@@ -112,3 +112,18 @@ Classes
     .. method:: getvalue()
 
         Get the current contents of the underlying buffer which holds data.
+
+.. class:: StringIO(alloc_size)
+.. class:: BytesIO(alloc_size)
```

Comment:
From a doc layout point of view, maybe it's better to list these with the other constructor signatures above.  Then the `getvalue()` method doesn't look strange attached to just the above, and not to this one.

Reply:
Surprisingly, this works pretty well, re: search, etc. Given that the intention is to deter a random reader from using them carelessly (by issuing a warning that they're not in CPython), that's a natural way to do it. (Because otherwise, the warning will look misplaced/cause unneeded alert).